data_models/data_refinery_models/migrations/0002_auto_20170727_2100.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
I thought that might be the case - why are there 2 migration files then though? Wouldn't that have reduced your migration files to just one?